src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -115,7 +124,8 @@ const ChatItemMyTextV2 = ({
   isContinuous = false,
   message = 'Ultrices neque op semper blahbla blahri mauris amet, penatibus. pi Amet, mollis quam venenatis di',
   status = ChatStatus.PENDING,
-  AnonymousImage = null
+  AnonymousImage = null,
+  chatType
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `chatType` prop is added but there's no default value provided. It's a good practice to provide default values for all props to prevent unexpected behavior if the prop is not passed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->